test/recipes/80-test_cmp_http.t

```diff
@@ -65,34 +65,40 @@ my $column;     # The column number of the expected result
 my $sleep = 0;  # The time to sleep between two requests
 my $server_fh;  # Server file handle
 
+sub subst_env {
+    my $val = shift;
+    return '""""' if $val eq "";
+    return $ENV{$1} if $val =~ /^\$\{ENV::(\w+)}$/;
+    return $val;
+}
+
 # The local $server_name variables below are among others taken as the name of a
 # sub-directory with server-specific certs etc. and CA-specific config section.
-
 sub load_config {
     my $server_name = shift;
     my $section = shift;
     my $test_config = $ENV{OPENSSL_CMP_CONFIG} // "$server_name/test.cnf";
-    open (CH, $test_config) or die "Cannot open $test_config: $!";
+    open (CH, $test_config) or die "Cannot open ".getcwd()."/$test_config: $!";
```

Comment:
No - this is not the right way to do it - you passed in a string to open - that is what should be logged.
If you want to know the current working directory then log that somewhere else - or change the logging line here to make it clear what was passed in and what is the current working directory.

Comment applies in both places you've done this.

Reply:
Thanks for you immediate comment and hint.
There is already 
```
    print "Current directory is ".getcwd()."\n";
```
so I've just removed the `getcwd()` invocations in the three failure reports where it applies.